core/src/main/java/cz/xtf/core/openshift/OpenShift.java

```diff
@@ -277,15 +279,20 @@ public KubernetesList createResources(List<HasMetadata> resources) {
     }
 
     public KubernetesList createResources(KubernetesList resources) {
-        return lists().create(resources);
+        return (KubernetesList) resourceList(resources.getItems()).create();
     }
 
     public boolean deleteResources(KubernetesList resources) {
-        return lists().delete(resources);
+        if (resourceList(resources.getItems()).delete() != null) {
+            return true;
+        } else {
+            return false;
+        }
     }
 
     public void loadResource(InputStream is) {
-        load(is).deletingExisting().createOrReplace();
+        load(is).withTimeout(30, TimeUnit.SECONDS).delete();
```

Comment:
do we need timeout? I know it is in example in migration guide, but I would let it use default.